main.go

```diff
@@ -35,14 +35,8 @@ func main() {
 			c.Logger().Error("Bind: ", err)
 			return c.String(http.StatusBadRequest, "Bind: "+err.Error())
 		}
-		var text *string
-		if newText, ok := task.Text(); ok {
-			text = &newText
-		}
-		var completed *bool
-		if newCompleted, ok := task.Completed(); ok {
-			completed = &newCompleted
-		}
+		text := &task.Text
+		completed := &task.Completed
```

Comment:
This does not work.

https://github.com/prisma/prisma-client-go/blob/33b5e2c53b2bcbbe4a7f6f8cac5b8653c6c4e521/docs/reference/15-if-present-methods.md#querying-dynamically
>but if a variable is nil, it means that that field should be ignored.

Reply:
😮 Thank you for letting me know!
🙇 I should read the reference...